src/sonic-config-engine/tests/sample_output/rules_for_dataacl.json

```diff
@@ -34,5 +34,12 @@
             "priority":9996
         },
         "OP":"SET"
+    },
+    {
+        "ACL_RULE_TABLE:dataacl:default_rule":{
+            "PACKET_ACTION":"DROP",
```

Comment:
all packet includes lldp, arp, lacp, they should not be dropped. It is better to limit to only ip.